Yearn-agents/PPSValues/src/agent.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { HandleBlock } from "forta-agent";
+import { TestBlockEvent } from "forta-agent-tools";
+import agent, { createFinding } from "./agent";
+import MockWeb3 from "./mock";
+
+describe("PPS ( Price per share ) agent", () => {
+  let handleBlock: HandleBlock;
+
+  beforeAll(() => {
+    let mockWeb3 = {
+      eth: {
+        Contract: MockWeb3.build_Mock(),
+      },
+    } as any;
+    handleBlock = agent.provideHandleFunction(mockWeb3 as any);
+  });
+
+  it("returns empty findings if pps increases", async () => {
+    const blockEvent = new TestBlockEvent();
+
+    const findings = await handleBlock(blockEvent);
+
+    expect(findings).toStrictEqual([]);
+  });
+
+  it("Returns findings if pps decreases", async () => {
+    const blockEvent = new TestBlockEvent();
+
+    const findings = await handleBlock(blockEvent);
+
+    expect(findings).toStrictEqual([
+      createFinding("1", "1.1", "Decrease in PPS", 1),
+    ]);
+  });
+
+  it("Returns findings if swift change in pps", async () => {
```

Comment:
test case for large change but has not reached the threshold

Reply:
can you tell what you mean by threshold. If there is 10% increase or more, the finding will be triggered, else nothing.